public/app/features/trails/interactions.ts

```diff
@@ -0,0 +1,45 @@
+import { reportInteraction } from '@grafana/runtime';
+
+const PREFIX = 'grafana_explore_metrics_';
+
+type ExploreMetricsInteractions =
+  // User clicked on the "add to filters" button to add the label=value to the filters variable from a label breakdown panel
+  | 'breakdown_panel_label_filter_added'
+  // User clicked on the "select" button to select a label from a label breakdown panel
+  | 'breakdown_panel_label_selected'
+  // User changed the breakdown layout
+  // {layout: string}
+  | 'breakdown_layout_changed'
+  // User clicked on link the overview page to look at the breakdown of a label
+  | 'label_breakdown_link_clicked'
+  // User opened metric explore from dashboard
+  | 'embedded_drawer_opened'
+  // User clicks on the button to start a new exploration
+  | 'exploration_started'
+  // User clicks on an existing trail of bookmark from the homepage
+  | 'exploration_reloaded'
+  // User clicks on the delete bookmark button on the home page
+  | 'bookmark_deleted'
+  // User changes metric explore settings
+  // {stickyMainGraph: boolean}
+  | 'settings_changed'
+  // User clicks on history nodes to navigate exploration history
+  // Do we care about which trail they were on? Each should have a unique id. From step, to step?
+  | 'history_step_clicked'
+  // User clicks on tab to change the action view
+  // {view: string}
+  | 'metric_action_view_changed'
+  // User toggles the bookmark button on the current Explore Metric state
+  // {setTo: boolean}
+  | 'bookmark_toggled'
+  // User opens current seleted metric and filter query in "Explore"
```

Comment:
do you mean the button highlighted?
![Screenshot by Dropbox Capture](https://github.com/grafana/grafana/assets/143556099/030ce284-0d3b-481e-95b1-2340feebad1e)

Reply:
Yes, for `'open_in_explore_clicked'`